src/controllers/exposures.js

```diff
@@ -21,33 +27,33 @@ async function exposuresPage (req, res) {
       partial: exposuresSetup,
       csrfToken: generateToken(res, req),
       nonce: res.locals.nonce,
-      fxaProfile: req.user?.fxa_profile_json
+      fxaProfile: user.fxa_profile_json
     }
     res.send(mainLayout(data))
     return
   }
 
+  const scanResults = await listScanResults(user.onerep_profile_id)
   /**
    * @type {MainViewPartialData<import('../views/partials/exposures-list').PartialParameters>}
    */
   const data = {
     partial: exposuresList,
     csrfToken: generateToken(res, req),
     nonce: res.locals.nonce,
-    fxaProfile: req.user?.fxa_profile_json
+    fxaProfile: req.user?.fxa_profile_json,
```

Comment:
Nit: should this use `user` variable as above? I know you didn't really modify this other than appending `,` but still :)

Reply:
Oh heh, yeah that's more consistent: 305a61afe